src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
```

Comment:
Remember about correct indentation between parent and child elements. The `<div class="container">` should be indented with 2 spaces inside the `<body>` tag.